src/mui/input/ReferenceArrayInput.js

```diff
@@ -182,8 +185,29 @@ export class ReferenceArrayInput extends Component {
                             label
                         )
                     }
-                    source={source}
-                    resource={resource}
+                >
+                    <span style={progessContainerStyle}>
```

Comment:
Why not extracting completely this component as `ReferenceLoadingProgress` and put the styles in it ?

Reply:
👍